### PR TITLE
This PR introduce an ability to set storage class with which volumes will be created

### PR DESCRIPTION
**Pull request by Kasen**
Also all resources will have prefix equal to helm installation name.

This change brеaks backward compatibility.

### CODE REVIEW

**Comment by VifleY** [comment]
`change brеakes backward` -> `change breaks backward`

**Comment by VifleY** [comment]
Commits message:Example:
If application installing with command:
helm install app ./helm-charts/tarantool-cartridge

then resources will have name:
ttq-routers-0

**Comment by VifleY** [comment]
Example in commit message is not obvious:
```
Example:
If application installing with command:
helm install app ./helm-charts/tarantool-cartridge

then resources will have name:
ttq-routers-0
```

Why the resources wil have `ttq` prefix?

**Comment by VifleY** [comment]
And last commit message seems to be broken (`volumes` is split among header and body)

File: helm-charts/tarantool-cartridge/templates/deployment.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ spec:
 apiVersion: tarantool.io/v1alpha1
 kind: Role
 metadata:
-  name: {{ .RoleName | replace "_" "" }}
+  name: "{{ $.Release.Name }}-{{ .RoleName | replace "_" "" }}"
```
Comment (DmitryTravyan) [inline helm-charts/tarantool-cartridge/templates/deployment.yaml:15]: It is best practice to use kebab-case or camelCase

**Pull request merged**